src/components/LearnerActivityTable/data/utils.js

```diff
@@ -0,0 +1,43 @@
+import { sendEnterpriseTrackEvent } from '@edx/frontend-enterprise-utils';
+
+export const trackDataTableEvent = ({
```

Comment:
[nit] Coverage is a bit low for this function. A individual unit test, or integration test with the hook using it would help increase the coverage and validate behavior.

Including it in the integration test in the learner activity tables a second benefit of producing a test that can be validated across other Datatable integration tests as well.

Reply:
`trackDataTableEvent` and `updateUrlWithPageNumber` are only being called (without using their return values) within the hook and component, respectively. Therefore, I will write a separate test for them in the utils.test.js file.